suite/tests/api/ir_x86.c

```diff
@@ -260,6 +265,16 @@ test_all_opcodes_2_avx512_evex(void *dc)
 #    undef INCLUDE_NAME
 }
 
+static void
+test_all_opcodes_2_avx512_evex_disp8(void *dc)
+{
+#    define INCLUDE_NAME "ir_x86_2args_avx512_evex.h"
+    memarg_disp = 0x80;
```

Comment:
IMHO this should have a comment explaining why it's separately testing 0x80, esp when the function name "disp8" seems to already be covered by 0x37.  Maybe the function name should be changed to be more clear.

Reply:
Done.